src/routers/courseItem.ts

```diff
@@ -22,13 +22,26 @@ cItemRouter.get('/:id', async (req, res) => {
 	}
 });
 
+// helper function converts string to number
+//   - should be updated to support fractions
+const numberify = (str: string) => {
+	if (str) {
+		return Number(str);
+	}
+};
+
 cItemRouter.post('/', async (req, res) => {
 	try {
 		const userID = req.header('userID');
 		if (userID) {
-			await update(req.body, userID);
+			const body = req.body;
+			body.owner = userID;
+			body.weight = numberify(body.weight);
+			body.grade = numberify(body.grade);
```

Comment:
So I realize we can make this a bit better by following some of the auggestions in this [SO answer](https://stackoverflow.com/questions/25983090/is-sanitizing-json-necessary). Specifically, do 

```js
const courseItem = {
    owner: userId;
    weight: numberify(body.weight);
    // other wanted properties ...
}
```

This helps us avoid adding properties that someone maliciously adds to the request etc.